jetty-core/jetty-server/src/main/config/etc/jetty-server-shutdown.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://jetty.org/configure_10_0.dtd">
+
+<!-- =============================================================== -->
+<!-- Jetty ServerShutdown                                            -->
+<!-- =============================================================== -->
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Call name="addBean">
+    <Arg>
+      <New id="ServerShutdown" class="org.eclipse.jetty.server.ServerShutdown">
```

Comment:
The `ServerShutdown` should be the subject of a `<Configure ` clause, so that there is only one for a set of XMLs
Then the jetty.xml can just add the bean by reference, and if it has not been created, then there is none.

Reply:
If it is Configured, it is then not started, as it cannot be a LifeCycle of it's own.
It is added as a bean to the server intentionally, so that the server can find it (if set already), or fallback to the ShutdownMonitor if it isn't configured yet.
Then the shutdown can be started on it's own, by the Server itself.